nerdlets/github-about/main.js

```diff
@@ -247,7 +246,13 @@ export default class GithubAbout extends React.PureComponent {
       documentId: 'repoUrl',
       document: { repoUrl }
     };
-    await EntityStorageMutation.mutate(mutation);
+    try {
+      await EntityStorageMutation.mutate(mutation);
+    } catch (error) {
+      if (!JSON.stringify(error).includes('Invariant Violation: 8')) {
```

Comment:
Is there any reason why we wouldn't throw all errors?

Reply:
@zstix this was a strange one that I may need some advice on, but basically an `Invariant Violation: 8` error was being thrown when trying to mutate with NerdGraph, which prevented the pack from refreshing (a manual page refresh was needed). This is an Apollo error which is not a module in the NerdPack, so it looks like this is being thrown from NerdGraph itself. I couldn't find any documentation on the specific error anywhere, even internally so it kinda stumped me. 

The other strange thing is that the mutation itself works, regardless of the error being thrown so it doesn't look like it's related to mutating data, so I just logged it for now while we figure out why this is being thrown and if it's a valid error or not